docs/developer-docs/smart-contracts/call/arguments.mdx

```diff
@@ -72,17 +72,27 @@ fn get_numbers(numbers: Vec<u32>) -> Vec<u32> {
 Azle refers to the `Vec` type to represent the equivalent of an `array` in Typescript. This is because `Vec` aligns with the [Candid](/docs/current/developer-docs/smart-contracts/candid/candid-concepts) type.
 
 ```typescript
-import { Canister, int32, Vec, query } from 'azle';
-let numbers = [0,1,2,3];
-export default Canister({
-    get_numbers: query([Vec(int32], Vec(int32), () => {
-        return numbers;
-    })
-});
+import { IDL, query } from 'azle';
+
+export default class {
+    @query([], IDL.Vec(IDL.Nat8))
+    getBlob(): Uint8Array {
+        return stringToBlob('hello');
+    }
+
+    @query([], IDL.Vec(IDL.Vec(IDL.Nat8)))
+    getBlobs(): Uint8Array[] {
+        return [stringToBlob('hello'), stringToBlob('world')];
+    }
+}
+
+function stringToBlob(string: string): Uint8Array {
+    return Buffer.from(string);
```

Comment:
Unfortunately I'm not sure Buffer is going to be a 1.0 feature. So it might only work if devs opt-in to experimental, with a command-line flag or dfx.json property.

Reply:
What is the best alternative?

Reply:
TextEncoder